.size-limit.js

```diff
@@ -7,19 +7,19 @@ module.exports = [
     name: 'Core - CDN',
     path: 'dist/legacy/rudder-analytics.min.js',
     gzip: true,
-    limit: '37.1 kB',
+    limit: '37.75 kB',
   },
   {
     name: 'All Integrations - CDN',
     path: 'dist/legacy/js-integrations/*.min.js',
     gzip: true,
-    limit: '432.7 kB',
+    limit: '462.23 kB',
   },
   {
     name: 'Core - NPM',
     path: 'dist/npm-lib/index.js',
     gzip: true,
-    limit: '37.1 kB',
+    limit: '37.67 kB',
```

Comment:
Please round off the limits to single precision.

Reply:
updated